Everyday 모듈 추가

starcat37 opened the pull request:
- Everyday 모듈을 추가했습니다.

starcat37 committed:
Add everydaymodule

starcat37 committed:
feat(Everyday): Add EverydayModule

susemeee commented:
이 모듈이 정상적으로 동작하기 위해서는, `Module#on_time` 함수가 주기적으로 호출이 되어야 합니다.
그래서 특정 시간이 지날 때마다 특정 함수를 호출하는 로직을 구현해야 하는데요, 이를 구현할 수 있는 방법은 여러가지가 있지만 일단 제가 써봤던 방법은 파이썬의 `sched` 모듈을 사용했던 것이에요.

[apscheduler](https://apscheduler.readthedocs.io/en/stable/)라는 라이브러리도 있으니 참고해 주세요.

starcat37 committed:
Schedule 모듈 추가

Schedule 모듈을 추가했고, 몇몇 오타를 수정하고, 화요일과 일요일 value의 내용을 수정했습니다.
그런데 이렇게 하면 모듈이 하루 종일 돌아가는 것 같은데, 이래도 괜찮을까요?

starcat37 commented:
Schedule 모듈 추가해서 Commit했습니다. 
그런데 이렇게 하면 모듈이 하루 종일 돌아가는 것 같은데, 이래도 괜찮을까요?

susemeee commented:
맞아요. 저렇게 하면 모듈이 하루 종일 돌아가기 때문에 괜찮지 않습니다. while 루프를 사용하지 않는 다른 방법이 있습니다.
하루종일 돌아가는 문제를 해결한 후, 모듈이 init될 때 scheduler를 실행하는 방식이 좋겠네요.

starcat37 committed:
remove duplication

starcat37 committed:
Merge branch 'susemeee:master' into master

starcat37 committed:
Exclude EverydayModule

starcat37 committed:
Merge branch 'master' of https://github.com/starcat37/yrumee

starcat37 commented:
- 대학원생을 중복으로 넣지 못하도록 하였습니다
- EverydayModule은 일단 client.py에서 제외했습니다
- 이렇게 하는 게 맞는 건지 모르겠어요
